test/Ocelot.IntegrationTests/CacheManagerTests.cs

```diff
@@ -1,16 +1,16 @@
-using CacheManager.Core;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
-using Newtonsoft.Json;
-using Ocelot.Administration;
-using Ocelot.Cache.CacheManager;
-using Ocelot.Configuration.File;
-using Ocelot.DependencyInjection;
-using Ocelot.Middleware;
-using System.Net;
-using System.Net.Http.Headers;
+using CacheManager.Core;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Newtonsoft.Json;
+using Ocelot.Administration;
+using Ocelot.Cache.CacheManager;
+using Ocelot.Configuration.File;
+using Ocelot.DependencyInjection;
+using Ocelot.Middleware;
+using System.Net;
+using System.Net.Http.Headers;

```

Comment:
😠

Reply:
Hopefully we'll solve it [here](https://github.com/ThreeMammals/Ocelot/pull/2156#discussion_r1792202279)

Reply:
Probably! Seems you need to rebase once again 😉